src/components/carousel/Carousel.vue

```diff
@@ -310,8 +310,7 @@ export default {
         },
         // handle drag event
         dragStart(event) {
-            if (!this.hasDrag ||
-                !event.target.draggable) return
```

Comment:
Without this condition, form elements (input, select, textarea, ...) does not work as expected inside the carousel.

Reply:
so maybe we need to play with `pointer-events`  of childs elements, i will check this later (had similar problem with child elements of custom drag n drop component).